client/components/Nav.js

```diff
@@ -1,5 +1,38 @@
 import React from 'react';
+import { Link } from 'react-router-dom';
 
 export const Nav = () => {
-  return <div />;
+  const navLinks = [
+    {
+      label: 'WebName',
+      to: '/',
+    },
+    {
+      label: 'AboutUs',
+      to: '/',
+    },
+    {
+      label: 'MeetTheTeam',
+      to: '/meetTheTeam',
+    },
+    {
+      label: 'ChallengesList',
+      to: '/challengesList',
+    },
+    {
+      label: 'Login/SignUp',
+      to: '/login',
+    },
+  ];
+  return (
+    <div className="nav nav-pills" style={{ marginBottom: '20px', marginTop: '10px' }}>
+      {navLinks.map(link => (
+        <div key={link.to} className="nav-item">
```

Comment:
In the navLinks you defined above, there are two link to goes to the same path '/', can we merge them or use different path name. Currently the following messages show up:
**react-dom.development.js:506 Warning: Encountered two children with the same key, `/`.** Keys should be unique so that components maintain their identity across updates. Non-unique keys may cause children to be duplicated and/or omitted — the behavior is unsupported and could change in a future version.

Reply:
@chem2016 @KristyHSC We just need to include an "id" field on line 30. to fix that error. I agree that we should change the url for one of the aboutus page.